client/src/pages/component-showcase.tsx

```diff
@@ -254,7 +254,7 @@ export default function ComponentShowcase() {
 
         <ComponentSection title="Bottom Nav">
           <div className="w-full">
-            <BottomNav isFixed={false} navIndex={0} />
+            <BottomNav isFixed={false} children={<></>} />
```

Comment:
maybe you can just delete this? and there are some conflicts with the main branch, please run `git merge origin main` and try to fix it
```suggestion
            <BottomNav isFixed={false} 
```